sdk/azidentity/interactive_browser_credential.go

```diff
@@ -125,10 +119,7 @@ func interactiveBrowserLogin(ctx context.Context, authorityHost string, opts *In
 	if err != nil {
 		return nil, err
 	}
-	redirectURL := opts.RedirectURL
-	if redirectURL == "" {
-		redirectURL = rs.Start(state.String(), opts.Port)
-	}
+	redirectURL := rs.Start(state.String())
```

Comment:
We really always need to run this local server?  This part was always unclear to me.

Reply:
Yes. The basic flow is:
1. open a browser to an AAD sign in page, providing a redirect URL in the query
1. user authenticates
1. AAD redirects to the given URL with an authorization code in the query
1. application redeems the authorization code for an access token

This flow was designed for web apps. To use it in local development, we need something local to receive the redirect and expose the auth code to our credential.